tests/testthat/test-summary-log.R

```diff
@@ -1,5 +1,7 @@
 context("Test creating summary logs")
 
+suppressPackageStartupMessages(library(dplyr))
```

Comment:
This line/commit from gh-444 is still sneaking in because the base was changed on github, the commit still needs to be dropped from the branch.

Reply:
ah - yeah - what is the best way of dropping that commit from the branch?

Reply:
Given it's the initial commit, you could

  * use interactive rebase (`git rebase -i origin/develop` and then drop the first line when the editor pops up)
  * a non-interactive rebase of just the top two commits: `git rebase --onto origin/develop HEAD~2`

Then force push.

---

Or just say "why bother" and add a new commit that reverts that `library` call :)

Reply:
i'm all about racking up those commit numbers, especially with how many @seth127 steals from me 😆

Reply:
Ha, in this case it's actually _my_ commit that would be dropped, but all right, forward-only it is.  I'll push a commit that drops/reverts the `library(dplyr)` call that initially prompted this PR, and I'll also push a commit to cleanup a few function calls whose alignment was thrown off by dd8578c.